language/stdlib/modules/VASP.move

```diff
@@ -174,6 +180,138 @@ module VASP {
         borrow_global_mut<ParentVASP>(parent_addr).compliance_public_key = new_key
     }
 
+    // **************** SPECIFICATIONS ****************
+
+    /// # Module specifications
+
+    spec module {
+        /// > TODO(emmazzz): verification is turned off because `Signature` module
+        /// > has not been specified. See issue #4666.
+        pragma verify = false;
+    }
+
+    spec module {
+        /// Returns true if `addr` is a VASP.
+        define spec_is_vasp(addr: address): bool {
```

Comment:
This could use spec_is_parent_vasp and spec_is_child_vasp, to reduce redundancy (suppose someone changes the test in spec_is_parent_vasp?)